src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -321,26 +321,36 @@ private bool CanInstallPackage()
         {
             if (DisplayRoots != null)
             {
-                foreach (PackageItem packageItem in DisplayRoots)
+                CheckIfFilesAreSelected(DisplayRoots);
+            }
+
+            return !_isInstalling && !NoFilesSelected;
```

Comment:
>_isInstalling [](start = 20, length = 13)

This doesn't change during the CheckIfFilesAreSelected call, right? If so, can you do this first?

Reply:
Refactored this code a bit. Could you please take a look at the new iteration and let me know what you think?

---
In reply to: [194554829](https://github.com/aspnet/LibraryManager/pull/126#discussion_r194554829) [](ancestors = 194554829)